src/app/systelab-components/modal/message-popup/message-popup-view.component.ts

```diff
@@ -26,6 +26,7 @@ export class MessagePopupViewContext extends SystelabModalContext {
 export class MessagePopupViewComponent implements ModalComponent<MessagePopupViewContext>, AfterViewInit {
 
 	@ViewChild('closeBtn', {static: false}) closeBtn: ElementRef;
+	@ViewChild('arrayBtn', {static: false}) arrayBtn: ElementRef;
```

Comment:
It is not only a reference to an element, but to an array of them

Reply:
In fact my tests indicate that it references the first item in the array, but maybe i'm wrong

Reply:
An improvement could be to define the button that can get the focus. This will be done in another PR